### DISCUSSION TITLE
artillery reporting

### DISCUSSION THREAD

**Discussion by rajutelus** [artilleryio/artillery#1946] [Artillery]
Hi all, is someone using any reporting tool for the artillery results?

**Comment by hassy** [comment]
Artillery comes with integrations with a bunch of monitoring/observability systems built in: https://www.artillery.io/docs/reference/extensions/publish-metrics

Or do you mean something like like Allure? (https://github.com/allure-framework)

There's https://www.artillery.io/cloud as well for visualizing and collaborating on Artillery test reports.
